tf6muxserver/mux_server_GetProviderSchema_test.go

```diff
@@ -192,6 +193,45 @@ func TestMuxServerGetProviderSchema(t *testing.T) {
 								},
 							},
 						},
+						ListResourceSchemas: map[string]*tfprotov6.Schema{ // TODO: Think of a good test
```

Comment:
It's a good test – we can drop the TODO comment 😃 

To make it a bit more realistic, we could:

* Remove the block – we'll try to avoid blocks in list resource schemas if possible
* Use a string attribute name that feels search-y. `filter`, `prefix`, etc.